aviary/interface/methods_for_level1.py

```diff
@@ -185,6 +185,12 @@ def _exec_level1(args, user_args):
     if isinstance(args.input_deck, list):
         args.input_deck = args.input_deck[0]
 
+    curr_dir = os.path.abspath(os.getcwd())
+    if args.outdir == curr_dir + os.sep + 'output':
```

Comment:
```suggestion
    if args.outdir == os.path.join(os.getcwd(), "output"):
```
it is probably safer to use the same command as before for the comparison, just in case one file path is relative and the other is absolute, but point to the same directory.